library/core/src/prelude/v1.rs

```diff
@@ -69,29 +68,26 @@ pub use crate::{
 #[doc(no_inline)]
 pub use crate::concat_bytes;
 
+// Do not `doc(no_inline)` so that they become doc items on their own
+// (no public module for them to be re-exported from).
+// Seems relevant for stable items (contrary to the following batches of items).
 #[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
-#[allow(deprecated, deprecated_in_future)]
-#[doc(no_inline)]
-pub use crate::macros::builtin::{
-    bench, global_allocator, test, test_case, RustcDecodable, RustcEncodable,
-};
-
+pub use crate::macros::builtin::{derive, global_allocator, test};
+//
 #[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
-#[doc(no_inline)]
-pub use crate::macros::builtin::derive;
+#[allow(deprecated)]
+pub use crate::macros::builtin::{bench, test_case, RustcDecodable, RustcEncodable};
```

Comment:
These imports can be merged with the import group above.
(Or `RustcDecodable/RustcEncodable` can be kept separate.)